cylc/rose/__init__.py

```diff
@@ -14,9 +14,48 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+
+Cylc Rose
+=========
+
 Cylc Rose is the bridge between Rose suite configurations and Cylc workflows.
 
 Cylc Rose is a replacement for the ``rose suite-run`` command (present in Rose
 versions 2019.01 and earlier). It reads the ``rose-suite.conf`` file and
 performs the required actions.
+
+Pre install
+===========
```

Comment:
I think this is a little over technical for the end-user who is not familiar with the underlying entry points. Could simplify this down into something a little more user-orientated, main things I think I would want to find as a user:

* Name the config sections cylc-rose works with and link through to the relevant Rose docs (can reference configs directly as the Rose docs are linked with intersphinx, prepend `rose:` to any references).
* Explain the `cylc-install` optional config.
* Give some uber-simple illustrative example.

Not over-important to get this perfect now though.

Reply:
This sounds good.